tools/integration-test/src/relayer/chain.rs

```diff
@@ -0,0 +1,362 @@
+use crossbeam_channel as channel;
+use ibc::core::ics02_client::client_consensus::{AnyConsensusState, AnyConsensusStateWithHeight};
+use ibc::core::ics02_client::client_state::{AnyClientState, IdentifiedAnyClientState};
+use ibc::core::ics02_client::events::UpdateClient;
+use ibc::core::ics02_client::misbehaviour::MisbehaviourEvidence;
+use ibc::core::ics03_connection::connection::IdentifiedConnectionEnd;
+use ibc::core::ics04_channel::channel::IdentifiedChannelEnd;
+use ibc::core::ics04_channel::packet::{PacketMsgType, Sequence};
+use ibc::query::QueryTxRequest;
+use ibc::{
+    core::ics02_client::header::AnyHeader,
+    core::ics03_connection::connection::ConnectionEnd,
+    core::ics03_connection::version::Version,
+    core::ics04_channel::channel::ChannelEnd,
+    core::ics23_commitment::commitment::CommitmentPrefix,
+    core::ics24_host::identifier::ChainId,
+    core::ics24_host::identifier::ChannelId,
+    core::ics24_host::identifier::{ClientId, ConnectionId, PortId},
+    events::IbcEvent,
+    proofs::Proofs,
+    query::QueryBlockRequest,
+    signer::Signer,
+    Height,
+};
+use ibc_proto::ibc::core::channel::v1::{
+    PacketState, QueryChannelClientStateRequest, QueryChannelsRequest,
+    QueryConnectionChannelsRequest, QueryNextSequenceReceiveRequest,
+    QueryPacketAcknowledgementsRequest, QueryPacketCommitmentsRequest, QueryUnreceivedAcksRequest,
+    QueryUnreceivedPacketsRequest,
+};
+use ibc_proto::ibc::core::client::v1::{QueryClientStatesRequest, QueryConsensusStatesRequest};
+use ibc_proto::ibc::core::commitment::v1::MerkleProof;
+use ibc_proto::ibc::core::connection::v1::QueryClientConnectionsRequest;
+use ibc_proto::ibc::core::connection::v1::QueryConnectionsRequest;
+use ibc_relayer::chain::handle::{ChainHandle, ChainRequest, Subscription};
+use ibc_relayer::chain::{HealthCheck, StatusResponse};
+use ibc_relayer::config::ChainConfig;
+use ibc_relayer::error::Error;
+use ibc_relayer::{connection::ConnectionMsgType, keyring::KeyEntry};
+
+use crate::types::tagged::*;
+
```

Comment:
This module does not export any items, so perhaps we should hide it from the docs. Otherwise it appears empty in the documentation:
<img width="735" alt="Screen Shot 2021-11-18 at 11 40 19" src="https://user-images.githubusercontent.com/106849/142400264-33ae8fbe-ba32-414c-b65b-f0ef9e47db08.png">

Reply:
I just added more documentation to it.